test/libsolidity/semanticTests/strings/empty_storage_string.sol

```diff
@@ -84,9 +84,9 @@ contract C {
 // g() -> 0x40, 0x60, 0, 0
 // h() -> 0x40, 0x60, 0, 0x1a, 38178759162904981154304545770567765692299154484752076569098748838215919075328
 // i() -> 0x40, 0x80, 0x1a, 38178759162904981154304545770567765692299154484752076569098748838215919075328, 0
-// j(string): 0x20, 0, "" -> 0x20, 0
+// j(string): 0x80, 0, "" -> FAILURE
 // k() -> 0x20, 0
-// l(string): 0x20, 0, "" -> 0x20, 0x40, 0x20, 0
+// l(string): 0x40, 0, "" -> 0x20, 0x40, 0x20, 0
```

Comment:
This looks unrelated to this PR. Why is it here?

Reply:
I guess it has to do with local tests and I must have overlooked when staging and committing. I am going to restore to the correct version.